src/match/controllers/create_match_controller.py

```diff
@@ -16,31 +16,39 @@ def process(self):
         match_ids = self._data.get("ids")
         if match_ids is None:
             error_msg = "POST body is misformatted"
-            return False, error_msg if self._return_status else failure_response(
-                error_msg, status.HTTP_400_BAD_REQUEST
+            return (
+                (False, error_msg)
+                if self._return_status
+                else failure_response(error_msg, status.HTTP_400_BAD_REQUEST)
```

Comment:
this is a bit strange

why can't we just return directly 
`failure_response(error_msg, self._return_status or status.HTTP_400_BAD_REQUEST)`

Reply:
yeah this part was created by @chalo2000 to work with the matching algorithm - I had to add parenthesis because it didn't work how he intended before. there's probably cleaner ways to do it but then we would have to change the matcher submodule too so I'm gonna leave it for now